src/mjolnir/convert_transit.cc

```diff
@@ -676,8 +760,12 @@ void AddToGraph(GraphTileBuilder& tilebuilder_transit,
         egress_node.set_stop_index(index);
         egress_node.set_timezone(timezone);
         egress_node.set_edge_index(tilebuilder_transit.directededges().size());
-        egress_node.set_connecting_wayid(egress.osm_way_id());
 
+        if (local_tile != nullptr && best_wayids[index] != std::numeric_limits<uint64_t>::max()) {
```

Comment:
if we dont do this the station is then not connected to the graph. we should at least log a warning in that case because it means people will not be able to get in or out at that station